tests/fixtures/tests.py

```diff
@@ -532,12 +533,13 @@ def test_loaddata_error_message(self):
             management.call_command('loaddata', 'invalid.json', verbosity=0)
             self.assertIn("Could not load fixtures.Article(pk=1):", cm.exception.args[0])
 
-    @ignore_warnings(category=UserWarning, message="No fixture named")
     def test_loaddata_app_option(self):
         """
         Verifies that the --app option works.
         """
-        management.call_command('loaddata', 'db_fixture_1', verbosity=0, app_label="someotherapp")
+        self.assertRaisesMessage(CommandError, "No fixture named 'db_fixture_1' found.",
```

Comment:
Please use the context manager version instead: `with self.assertRaisesMessage(CommandError, "No fixture named 'db_fixture_1' found."):`